### ISSUE TITLE
Use Black for formating

### ISSUE THREAD

**Issue by themperek** [SiLab-Bonn/basil#113]
Should we reformat everything with [Black](https://github.com/psf/black)?
With option `-l 160`?

@DavidLP @laborleben

**Comment by DavidLP** [comment]
Good idea. Code base is rather complete. I tried Black some years ago and it gave not really nice results. Maybe this has changed.
